api/v1alpha1/contour_types.go

```diff
@@ -58,6 +42,26 @@ type ContourList struct {
 	Items           []Contour `json:"items"`
 }
 
+// ContourSpec defines the desired state of Contour.
+type ContourSpec struct {
+	// replicas is the desired number of Contour replicas. If unset,
+	// defaults to 2.
+	//
+	// +optional
+	// +kubebuilder:default=2
+	// +kubebuilder:validation:Minimum=0
```

Comment:
@jpeach should a maximum be specified?

Reply:
I think leaving the maximum open is fine, but should we make the minimum 1?

Reply:
I can see the need to scale down a deployment to 0.... maybe something like an upgrade a transition apps between contour instances.

Reply:
ok that sounds reasonable